test/parse_test.go

```diff
@@ -16,5 +16,20 @@ func TestParseCommandSplit(t *testing.T) {
 	if len(cmd.Tokens) != 4 {
 		t.Errorf(formatError(commandWithRandomSpaces, "incorrectly tokenized command"))
 	}
-	log.Println(cmd.RawCommandString)
+	assert.Equal(t, cmd.RawCommandString, "git commit -m there   are random spaces in here", "Parsing removes unnecessary spaces in the raw command")
+}
+
+func TestParseCommandFlags(t *testing.T) {
```

Comment:
Don't know if we need this because we are refactoring flag parsing. We should just have test cases for the whole parsing of commands. And check if the command object matches what it should be